tests/jest/unitTest/UI_layer/EditOption.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { EditOption } from "../../../../src/Components/utils/EditOption";
```

Comment:
```suggestion
import { EditOption } from "@components/utils/EditOption";
```
could use @components here